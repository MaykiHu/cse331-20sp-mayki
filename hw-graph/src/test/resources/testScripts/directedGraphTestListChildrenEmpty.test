#Create a graph
CreateGraph graph1

#Create a node
CreateNode 1

#Lists children of node n1 in graph1
ListChildren graph1 n1