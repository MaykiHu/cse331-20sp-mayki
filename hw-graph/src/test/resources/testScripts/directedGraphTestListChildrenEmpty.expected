#Create a graph
created graph graph1

#Create a node
created node n1

#Lists children of node n1 in graph1
the children of n1 in graph1 are: